integration/src/main/java/scripts/testscripts/DatasetOperations.java

```diff
@@ -107,7 +132,19 @@ private void exportUserJourney(
     // Create workspace dataset
     var request =
         new CreateDatasetRequest()
-            .catalogEntry(createMetadata("export-test-dataset").toString())
+            .catalogEntry(
+                createMetadata(
+                        "crud",
+                        "description",
+                        "creator",
+                        "date",
+                        new ArrayList(),
```

Comment:
Unless you need to create a modifiable list, you should use `List.of()` instead of `new ArrayList<>()` to create an empty list.